src/setupTests.js

```diff
@@ -0,0 +1,18 @@
+jest.mock( 'process', () => ( { cwd: () => 'cwd' } ) );
```

Comment:
For the custom matcher, I'm all in favor of adding it here. `process.cwd`, however, I would not put here. This is basically changing system-specifics here, for all tests. And also, in your tests you rely on the fact that `cwd` returns the string `'cwd'`. So, personally, I would have this mock twice, in each file. It's just a single line of code.

Reply:
:+1: